modules/schematics/src/effect/index.spec.ts

```diff
@@ -50,6 +50,44 @@ describe('Effect Schematic', () => {
     ).toBeGreaterThanOrEqual(0);
   });
 
+  it('should create an effect class which has a "loadFoos$" property if feature is set', () => {
```

Comment:
This should be `should create an effect that describes a source of actions within a feature`.